doc/primer.rst

```diff
@@ -74,17 +75,18 @@ Or like this : ::
     part = part.cutThruAll()
 
 .. note::
-  While the first code style is what people default to, it's important to note that when you write your code like this it's equivalent as writting it on a single line.
-  It's then more difficult to debug as you cannot visualize each operation step by step, which is a functionality that is provided by the CQ-Editor debugger for example.
+  While the chained code style is succinct and intuitive and may be what people default to, it's important to note that all operations take place within a single statement (even if the code spans multiple lines).
+  This chained style be more difficult to debug using tools that allow a user to step statement-by-statement through the code (such as the CQ-Editor), because operations will occur simultaneously as part of a single statement.
```

Comment:
```suggestion
  This chained style can be more difficult to debug using tools that allow a user to step statement-by-statement through the code (such as the CQ-Editor), because operations will occur simultaneously as part of a single statement.
```